document/travis-deploy.sh

```diff
@@ -9,18 +9,10 @@ TARGET_BRANCH="gh-pages"
 
 function doCompile {
   # TODO(littledan): Integrate with document/deploy.sh
-  cd document/js-api
+  cd document
```

Comment:
Nit: `(cd document; make)`, removing the `cd ..`

Reply:
Fixed